test/e2e/atlas/helper_test.go

```diff
@@ -292,6 +292,17 @@ func deleteClusterForProject(projectID, clusterName string) error {
 		return fmt.Errorf("error deleting cluster %w: %s", err, string(resp))
 	}
 
+	// this command will fail with 404 once the cluster is deleted
+	// we just need to wait for this to close the project
+	_ = watchCluster(projectID, clusterName)
+	return nil
```

Comment:
wdyt about something like?
```suggestion
	if watchCluster(projectID, clusterName) != nil {
	      return nil
	}
	return errors.New("cluster is still active after delete attempt")
```

Reply:
I'm not too sure how to reproduce the situation you're suggesting, do we have that scenario already?

Reply:
I was looking at the watch command and there's no timeout for it (this was one of the possible scenarios to reproduce this), and also the watch status is printed for both success / error cases (so no need for additional logs here).